### PR TITLE
kelly/fix_dob_not_showing_in_android

### PR DESCRIPTION
**Pull request by kellybinary**

### CODE REVIEW

File: src/javascript/app/common/form_manager.js [hunk 1]
```diff
@@ -54,8 +54,6 @@ const FormManager = (() => {
                     value = val || '';
                     if (field.value) {
                         value = typeof field.value === 'function' ? field.value() : field.value;
-                    } else if (field.$.attr('data-picker') === 'native') {
```
Comment (negarn) [inline src/javascript/app/common/form_manager.js]: won't removing this break it for other pages that are using native date picker? e.g. trade page, self exclusion, etc

Comment (kellybinary) [reply src/javascript/app/common/form_manager.js]: was there any reason we're not getting value from `data-value` attribute for datepicker?

Comment (negarn) [reply src/javascript/app/common/form_manager.js]: I don't remember, but it's good if we can test all scenarios if we're removing it.

Comment (negarn) [reply src/javascript/app/common/form_manager.js]: I have added a checklist to the card, will approve it for now

Comment (kellybinary) [reply src/javascript/app/common/form_manager.js]: ok, hopefully, nothing breaks 🙏 😄

**Pull request merged**